test/common/upstream/cluster_manager_impl_test.cc

```diff
@@ -242,10 +242,7 @@ TEST_F(ClusterManagerImplTest, MaxClusterName) {
   }
   )EOF";
 
-  EXPECT_THROW_WITH_MESSAGE(create(parseBootstrapFromJson(json)), Json::Exception,
-                            "JSON at lines 4-6 does not conform to schema.\n Invalid schema: "
-                            "#/properties/name\n Schema violation: maxLength\n Offending "
-                            "document key: #/name");
+  EXPECT_THROW(create(parseBootstrapFromJson(json)), EnvoyException);
```

Comment:
Why does this throw now? What's the new error?

Reply:
This was always throwing an error. Earlier, it threw a JSON schema error with some message like "Invalid #/properties/name...." .. I merely changed this to use EnvoyException, as we are no longer validating the length using JSON schema.

In other words, UX wise, you would still get an exception when max length is exceeded, except it will be worded differently.

Reply:
Right, please keep EXPECT_THROW_WITH_MESSAGE and switch it to the new error.